hack/test-utils.sh

```diff
@@ -18,6 +18,13 @@ source $(dirname "${BASH_SOURCE[0]}")/utils.sh
 
 # RESTART_WAIT_PERIOD is the period to wait before restarting containerd.
 RESTART_WAIT_PERIOD=${RESTART_WAIT_PERIOD:-10}
+CONTAINERD_CONFIG=""
+
+# Use a configuration file for containerd.
+CONTAINERD_CONFIG_FILE=${CONTAINERD_CONFIG_FILE:-none}
```

Comment:
Let's use `""` as default. Although unlikely, but is possible that there is a `none` file.

And then you can check `-z` or `-f`.